src/theme.ts

```diff
@@ -124,38 +144,129 @@ declare module '@mui/material' {
   }
 }
 
-const theme = createTheme({
-  palette: {
-    primary: gordonColors.primary.blueShades,
-    secondary: gordonColors.primary.cyanShades,
-    error: gordonColors.secondary.redShades,
-    success: gordonColors.secondary.greenShades,
-    warning: gordonColors.secondary.yellowShades,
-    info: gordonColors.primary.cyanShades,
-    neutral: gordonColors.neutral,
-  },
-  typography: {
-    fontFamily: ['Gotham SSm 7r', 'Gotham SSm A', 'Gotham SSm B', 'sans-serif'].join(','),
-  },
-  components: {
-    MuiTooltip: {
-      styleOverrides: {
-        tooltip: {
-          backgroundColor: gordonColors.neutral.darkGray,
+/*const theme = createTheme({
+ palette: {
+   primary: gordonColors.primary.blueShades,
+   secondary: gordonColors.primary.cyanShades,
+   error: gordonColors.secondary.redShades,
+   success: gordonColors.secondary.greenShades,
+   warning: gordonColors.secondary.yellowShades,
+   info: gordonColors.primary.cyanShades,
+   neutral: gordonColors.neutral,
+ },
+ typography: {
+   fontFamily: ['Gotham SSm 7r', 'Gotham SSm A', 'Gotham SSm B', 'sans-serif'].join(','),
+ },
+ components: {
+   MuiTooltip: {
+     styleOverrides: {
+       tooltip: {
+         backgroundColor: gordonColors.neutral.darkGray,
+       },
+     },
+   },
+   MuiButton: {
+     styleOverrides: {
+       containedPrimary: {
+         color: gordonColors.neutral.grayShades.A100,
+       },<ModeSwitcher />
+       containedSecondary: {
+         color: gordonColors.neutral.grayShades.A100,
+       },
+     },
+   },
+ },
+}); */
```

Comment:
```suggestion
/*const theme = createTheme({
 palette: {
   primary: gordonColors.primary.blueShades,
   secondary: gordonColors.primary.cyanShades,
   error: gordonColors.secondary.redShades,
   success: gordonColors.secondary.greenShades,
   warning: gordonColors.secondary.yellowShades,
   info: gordonColors.primary.cyanShades,
   neutral: gordonColors.neutral,
 },
 typography: {
   fontFamily: ['Gotham SSm 7r', 'Gotham SSm A', 'Gotham SSm B', 'sans-serif'].join(','),
 },
 components: {
   MuiTooltip: {
     styleOverrides: {
       tooltip: {
         backgroundColor: gordonColors.neutral.darkGray,
       },
     },
   },
   MuiButton: {
     styleOverrides: {
       containedPrimary: {
         color: gordonColors.neutral.grayShades.A100,
       },<ModeSwitcher />
       containedSecondary: {
         color: gordonColors.neutral.grayShades.A100,
       },
     },
   },
 },
}); */
```
Reminder to remove commented code if unneeded